diff --git a/Sources/LinuxI2CDev/I2CUmbrella.h b/Sources/LinuxI2CDev/I2CUmbrella.h
index 1e0610d..66d406e 100644
--- a/Sources/LinuxI2CDev/I2CUmbrella.h
+++ b/Sources/LinuxI2CDev/I2CUmbrella.h
@@ -1,5 +1,8 @@
 #ifdef __APPLE__
 // the I2C headers are fairly self contained; it is possible to stub them here
+#include "/Users/kit/tmp/include/linux/types.h"
+#include "/Users/kit/tmp/include/linux/i2c.h"
+#include "/Users/kit/tmp/include/linux/i2c-dev.h"
 #else
 #include <linux/types.h>
 #include <linux/i2c.h>
